src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/TransactionServiceImpl.java

```diff
@@ -37,4 +39,14 @@ public Integer saveTransaction(final TransactionDto transactionDto) {
 		this.transactionInputValidator.validate(transactionDto);
 		return this.transactionService.saveTransaction(transactionDto);
 	}
+
+	@Override
+	public List<TransactionStatus>  getAllTransactionStatuses() {
```

Comment:
Even when statuses is a valid plural, it is uncommon. I think we should use status for plural

Reply:
Done